float32 charge
float32 battery1_voltage
float32 battery2_voltage
float32 current

